src/features/languages/generated/client/client.json

```diff
@@ -861,6 +868,8 @@
         "form.field.label.childDateOfBirth": "Date of birth",
         "form.field.label.childFamilyName": "Last name",
         "form.field.label.childFirstNames": "First name(s)",
+        "form.field.label.childFamilyNameEng": "Last Name(s) in English",
+        "form.field.label.childFirstNamesEng": "First Name(s) in English",
```

Comment:
This is not right.  Why do we need this @sadmananik ?

Reply:
Removed it.